src/domain/Modals/Modal/subcomponents/style.ts

```diff
@@ -79,9 +87,13 @@ const StyledModalControls = styled.div`
   display: flex;
 `
 
-const StyledHeaderControls = styled.div`
+const StyledHeaderRightComponent = styled.div`
   display: inline-flex;
   margin-left: 12px;
+
+  @media only screen and (max-width: ${Variables.Breakpoint.breakpointTablet}px) {
```

Comment:
Should this just not render this content on mobile instead of hiding it with CSS?